### PR TITLE
Remove panel closing transition

### PR DESCRIPTION
**Pull request by yucheng11122017**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [X] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

Resolves https://github.com/MarkBind/markbind/issues/1836
Duplicate of #1965

**Overview of changes:**
Removes all closing transition for panels

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
```Remove closing transition for panels```

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [X] Added tests for bug fixes or features
- [X] Linked all related issues
- [X] No unrelated changes

### CODE REVIEW

File: packages/vue-components/src/panels/PanelBase.js [hunk 1]
```diff
@@ -135,25 +135,17 @@ export default {
...
-        this.$refs.panel.style.maxHeight = `${this.$refs.panel.scrollHeight}px`;
-
         requestAnimationFrame(() => {
           // To enable behaviour of auto window scrolling during panel collapse
           if (this.$el.getBoundingClientRect().top < 0) {
             const headerHeight = jQuery('header[sticky]').height() || 0;
-            jQuery('html').animate({
-              scrollTop: window.scrollY + this.$el.getBoundingClientRect().top - headerHeight - 3,
-            }, 500, 'swing');
+            window.scrollTo(0, window.scrollY + this.$el.getBoundingClientRect().top - headerHeight - 3);
```
Comment (ang-zeyu) [inline packages/vue-components/src/panels/PanelBase.js:142]: It looks much better than the jQuery one, but oddly it still animates, contrary to the mdn documentation.. Is that intended?

Comment (yucheng11122017) [reply packages/vue-components/src/panels/PanelBase.js:142]: Hi @ang-zeyu, could I check what do you mean by it still animates? 
It seems to not be animated (except for the buttons) on my end

https://user-images.githubusercontent.com/77204346/221115277-dd68aacd-2e9d-4eab-a556-32b167ec2c09.mp4

Comment (ang-zeyu) [reply packages/vue-components/src/panels/PanelBase.js:142]: This line ^ (`window.scrollTo`)

Comment (ang-zeyu) [reply packages/vue-components/src/panels/PanelBase.js:142]: paneltransition

Comment (yucheng11122017) [reply packages/vue-components/src/panels/PanelBase.js]: Ah I see! 
@ang-zeyu is this the desired behaviour? 

https://user-images.githubusercontent.com/77204346/221341761-4f7d6a23-20b2-4d75-960d-1883cbe78976.mp4

**Review by ang-zeyu — APPROVED** [review]

File: packages/vue-components/src/panels/PanelBase.js (hunk 1 above)
Comment (ang-zeyu) [reply packages/vue-components/src/panels/PanelBase.js]: Yes

**Pull request merged**